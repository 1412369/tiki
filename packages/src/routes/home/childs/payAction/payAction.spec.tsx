import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PayAction from './payAction';
describe('snapshot PayAction', function() {
  it('Should render Payction without crashing', function() {
    const wrapper = renderer.create(<PayAction onPaymentClick={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
