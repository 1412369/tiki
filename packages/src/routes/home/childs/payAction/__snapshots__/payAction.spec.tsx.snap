// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot PayAction Should render Payction without crashing 1`] = `
<div
  className="sc-bdVaJa imRcMU"
>
  <button>
    Combo
  </button>
  <button
    onClick={[Function]}
  >
    Thanh to√°n
  </button>
</div>
`;
